controller/utils.go

```diff
@@ -0,0 +1,46 @@
+package controller
+
+import (
+	longhorn "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta2"
+	"github.com/longhorn/longhorn-manager/types"
+)
+
+func hasReplicaEvictionRequested(rs map[string]*longhorn.Replica) bool {
+	for _, r := range rs {
+		if r.Status.EvictionRequested {
+			return true
+		}
+	}
+
+	return false
+}
+
+func isVolumeMigrating(v *longhorn.Volume) bool {
```

Comment:
Suggest moving to volume_controller.go.

Reply:
Same as https://github.com/longhorn/longhorn-manager/pull/1541#discussion_r1193369051